src/main/java/org/interledger/codecs/oer/OerIA5StringCodec.java

```diff
@@ -41,15 +39,15 @@ public OerIA5String read(
   }
 
   @Override
-  public void write(
-      final CodecContext context, final OerIA5String instance, final OutputStream outputStream
-  ) throws IOException {
+  public void write(final CodecContext context, final OerIA5String instance,
+      final OutputStream outputStream) throws IOException {
 
     Objects.requireNonNull(context);
     Objects.requireNonNull(instance);
     Objects.requireNonNull(outputStream);
 
-    final byte[] data = instance.getValue().getBytes();
+    final byte[] data = instance.getValue()
+        .getBytes();
```

Comment:
It took me ages to get my formatter to do this! 😄 

This formatting is how we like the builder's to look but it seems it's impossible to have a rule that works for builders but not for other chained method calls.

Maybe need to get the formatter to ignore and leave it to use mortals to do correctly?